radicle-cli/src/terminal/args.rs

```diff
@@ -48,6 +49,36 @@ pub trait Args: Sized {
     fn from_args(args: Vec<OsString>) -> anyhow::Result<(Self, Vec<OsString>)>;
 }
 
+#[derive(Debug)]
+pub enum NodeId {
+    Did(prelude::Did),
+    Nid(prelude::NodeId),
+}
```

Comment:
Correct, both `Did` and `NodeId` are aliases for `PublicKey`.  However when node alias'es are supported, the type will no longer be redundant.

We can type alias it to prelude::NodeID, or provide a simple parse method.

Reply:
To me sounds like we are overcomplicating things with this enum stuff, we could just store the public key of the node inside the `Add` and `Rm` operations and parse the `nid` and `did` in a public key.

I did see the reason to add this additional indirection, just because the `track` cli do in this way?

Reply:
My 2c is that we only use `Did` in the `add` and `rm` commands

Reply:
> > My 2c is that we only use Did in the add and rm commands
> 
> What about consistency with other commands? whichever we choose it should be the same for `rad track` no?

Aye, I think we should be using DID as much as possible -- unless NID really does make sense, but that's a judgement call. So maybe in this case, as you're pointing out, it should be NID because we store NIDs for `rad track` and not DIDs. In fact, other DID methods might allow their own way of specifying an alias within their DID document.

Reply:
>Aye, I think we should be using DID as much as possible

Sorry for the stupid question but why? 

To me sounds that hide the `did` and `nid` under the command are pretty good as usability? or maybe the `did` has a different meaning and does not make sense?

Reply:
Ok we did not converge to a final solution, so I clean up the args and parse it as pubkey, so as discussed in https://radicle.zulipchat.com/#narrow/stream/369277-heartwood/topic/accept.20only.20.60nid.60.20in.20.60rad.20remote.20*.60/near/350040100 for the moment let accept both in the args and iterate it through it.

I think it is better to start to use it and then see how we feel about it